src/python/tests/core/local/butler/deploy_test.py

```diff
@@ -17,7 +17,6 @@
 import json
```

Comment:
Probably revert these deploy_test, these need to fixed with deployment changes in another Cl by @oliverchang